README.md

```diff
@@ -113,8 +121,82 @@ E1 - VME based encoding
+
+## Supported Decode Features
+
+Output format and max resolution support table like below.
+
+2k=2048, 4k=4096x4096, 8k=8192x8192, 16k=16384x16384
+
+| Codec      | Type     | BDW  | SKL  | BXT/APL | KBL  | CFL  | CNL  | ICL            |
+|------------|----------|------|------|---------|------|------|------|----------------|
+| AVC        |Output    | NV12 | NV12 |  NV12   | NV12 | NV12 | NV12 | NV12           |
+|            |Resolution| 4k   | 4k   |  4k     | 4k   | 4k   | 4k   | 4k             |
+| MPEG-2     |Output    | NV12 | NV12 |  NV12   | NV12 | NV12 | NV12 | NV12           |
+|            |Resolution| 2k   | 2k   |  2k     | 2k   | 2k   | 2k   | 2k             |
+| VC-1       |Output    | NV12 | NV12 |  NV12   | NV12 | NV12 | NV12 | NV12           |
+|            |Resolution| 4k   | 4k   |  4k     | 4k   | 4k   | 4k   | 4k             |
+| JPEG*      |Resolution| 16k  | 16k  |  16k    | 16k  | 16k  | 16k  | 16k            |
+| VP8        |Output    | NV12 | NV12 |  NV12   | NV12 | NV12 | NV12 | NV12           |
+|            |Resolution| 4k   | 4k   |  4k     | 4k   | 4k   | 4k   | 4k             |
+| HEVC 8bit  |Output    |      | NV12 |  NV12   | NV12 | NV12 | NV12 | NV12/YUY2/AYUV |
+|            |Resolution|      | 8k   |  8k     | 8k   | 8k   | 8k   | 8k             |
+| HEVC 10bit |Output    |      |      |  P010   | P010 | P010 | P010 | P010/Y210/Y410 |
+|            |Resolution|      |      |  8k     | 8k   | 8k   | 8k   | 8k             |
+| VP9  8bit  |Output    |      |      |  NV12   | NV12 | NV12 | NV12 | NV12/AYUV      |
+|            |Resolution|      |      |  4k     | 4k   | 4k   | 4k   | 8k             |
+| VP9  10bit |Output    |      |      |         | P010 | P010 | P010 | P010/Y410      |
+|            |Resolution|      |      |         | 8k   | 8k   | 8k   | 8k             |
+
+\* Jpeg supports output format: NV12/411P/422H/422V/444P/BGRP/RGBP/YUY2/ARGB
+
+
+## Supported Encode Features
+
+### Low Power Encoding:
+
+Output format supports NV12. Input format and max resoultuion support table like below. 
+
+JPEG encoding supports input format NV12/YUY2/UYVY/AYUV/ABGR/Y8 and output format YUV400/YUV420/YUV422H_2Y/YUV444/RGB24. 
+
+4k=4096x4096, 16k=16384x16384
+
+| Codec      | Type     | BDW  | SKL  | BXT/APL | KBL  | CFL  | CNL  | ICL**   |
+|------------|----------|------|------|---------|------|------|------|---------|
+| AVC        |Input     |      |      |  NV12   | More*| More*|      | Pending |
+|            |Resolution|      |      |  4k     | 4k   | 4k   |      | Pending |
+| JPEG       |Resolution|      | 16k  |  16k    | 16k  | 16k  | 16k  | Pending |
+|HEVC 8/10bit|Input     |      |      |         |      |      |      | Pending |
+|            |Resolution|      |      |         |      |      |      | Pending |
+| VP9 8/10bit|Input     |      |      |         |      |      |      | Pending |
+|            |Resolution|      |      |         |      |      |      | Pending |
```

Comment:
You write "pending", but CQP should work right? only BRC modes don't work - I would emphasize this somehow. Maybe changed "pending" to "CQP (BRC modes - pending)"

Reply:
SSEU kernel issue impact all kernel related encoding including CQP TU1.

Reply:
SSEU can't impact low power encoding since it is done on VDENC, right?. And this paragraph is about low power. In my mind low power can only be affected by the absent HuC, but CQP should work fine.

Reply:
No, TU1 is done by kernel. VDEnc is blocked by Huc loading issue.